docs/kyma-environment-broker/03-17-runtime-reconciler.md

```diff
@@ -0,0 +1,33 @@
+# Runtime Reconciler
+
+Runtime Reconciler is an application that performs reconciliation tasks on SAP BTP, Kyma runtimes (SKRs).
+
+## Overview
+
+Currently, there is one task for Runtime Reconciler. It reconciles BTP Manager Secrets on Kyma runtimes. It can do it in two ways: 
```

Comment:
```suggestion
Currently, there is one task for Runtime Reconciler. It reconciles BTP Manager Secrets on Kyma Runtimes. It can do it in two ways: 
```